core/src/main/java/hudson/model/DirectoryBrowserSupport.java

```diff
@@ -520,7 +552,7 @@ private int dirRank(VirtualFile f) {
                                 break;
                             f = sub.get(0);
                             relPath += '/'+Util.rawEncode(f.getName());
-                            l.add(new Path(relPath,f.getName(),true,0, f.canRead()));
+                            l.add(new Path(relPath,f.getName(),true, f.length(), f.canRead(), f.lastModified()));
```

Comment:
What motivates this change?

Reply:
it is the same value (0) but there is not a magic number

```
    /**
     * Gets the file length.
     * @return a length, or 0 if inapplicable (e.g. a directory)
     * @throws IOException if checking the length failed
     */
    public abstract long length() throws IOException;
```